wicket-core/src/main/java/org/apache/wicket/markup/html/pages/wicket-browser-info.js

```diff
@@ -58,6 +58,8 @@
 					info.screenHeight = window.screen.height;
 					info.screenColorDepth = window.screen.colorDepth;
 				}
+				var jsTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
According to https://caniuse.com/#feat=mdn-javascript_builtins_intl_datetimeformat_resolvedoptions this would fail on Android devices. Let's add some `isDefined` checks.